deployment/terraform/reloader.tf

```diff
@@ -0,0 +1,48 @@
+# Reloader is a Kubernetes controller that watches changes in ConfigMap and
+# Secrets and ensures that any Pods using them are restarted.
+# https://github.com/stakater/Reloader
+
+resource "kubernetes_namespace" "reloader" {
```

Comment:
Alternative to reloader, we can also handle reloading the file in the application. i.e. re-reading the file once in a while.